server/src/main/resources/db/migration/postgresql/V3_0__Remove_Scope_Permissions_Unique_Index.sql

```diff
@@ -1,3 +1,6 @@
+-- The original index from V1_1 may have been established as a constraint.  This will ensure the constraint is removed
+-- before attempting to drop the index.
+ALTER TABLE scope_permissions DROP CONSTRAINT IF EXISTS scope_permissions_scope_address_grantee_address_granter_address;
```

Comment:
I made this migration very poorly to begin with.  These changes will safely ensure that any scenario that arises due to the unique index established in V1_1 will be removed before migration V3_1 that adds the new index that includes the `grant_id` column.